src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -239,6 +247,38 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return _defaultAppearanceViewModel;
     }
 
+    bool ProfileViewModel::HasUnfocusedAppearance()
+    {
+        return _profile.HasUnfocusedAppearance();
+    }
+
+    void ProfileViewModel::CreateUnfocusedAppearance(const Windows::Foundation::Collections::IMapView<hstring, Model::ColorScheme>& schemes,
+                                                     const IHostedInWindow& windowRoot)
+    {
+        _profile.CreateUnfocusedAppearance();
+
+        _unfocusedAppearanceViewModel = winrt::make<implementation::AppearanceViewModel>(_profile.UnfocusedAppearance().try_as<AppearanceConfig>());
+        _unfocusedAppearanceViewModel.Schemes(schemes);
+        _unfocusedAppearanceViewModel.WindowRoot(windowRoot);
+
+        _NotifyChanges(L"UnfocusedAppearance");
+        _NotifyChanges(L"HasUnfocusedAppearance");
```

Comment:
```suggestion
        _NotifyChanges(L"UnfocusedAppearance", L"HasUnfocusedAppearance");
```
nit: `_NotifyChanges` is nice because it can just take multiple for you